src/core/hle/service/hid/controllers/npad.cpp

```diff
@@ -502,15 +514,47 @@ void Controller_NPad::SetNpadMode(u32 npad_id, NPadAssignments assignment_mode)
 
 void Controller_NPad::VibrateController(const std::vector<u32>& controller_ids,
                                         const std::vector<Vibration>& vibrations) {
-    LOG_DEBUG(Service_HID, "(STUBBED) called");
 
     if (!can_controllers_vibrate) {
         return;
     }
     for (std::size_t i = 0; i < controller_ids.size(); i++) {
-        std::size_t controller_pos = NPadIdToIndex(static_cast<u32>(i));
+        const u8 byte0 = controller_ids[i] % 256;         // possible motor direction?
```

Comment:
```suggestion
        const u8 byte0 = static_cast<u8>(controller_ids[i] % 256);         // possible motor direction?
```

ditto for the other byte expressions, below this, otherwise this will cause type truncation warnings.